src/pages/product/link.tsx

```diff
@@ -167,10 +169,10 @@ export default function LinkApp(props: ServerSideProps) {
               <div className={classes.gradientBox} />
             </Flex>
             <Title className={classes.heading} order={3}>
-              Keep track of activities
+              Keep track of your activities
             </Title>
             <Text className={classes.copy}>
-              See the history of all the models you have added to your Stable Diffusion & Flux model library.
+              View the history of all models you've added to your local model library.
```

Comment:
You can't add special characters (such as `'`) in HTML. Need to add the HTML version of it:

```suggestion
              View the history of all models you&rsquo;ve added to your local model library.
```